springer-mathphys-author-date.csl

```diff
@@ -114,7 +115,7 @@
       <key macro="author-in-citation"/>
       <key macro="issued-sort" sort="ascending"/>
     </sort>
-    <layout suffix=".">
+    <layout>
```

Comment:
None of the references have a period at the end or just not the ones with DOI? For the latter, we'd need to find a better solution. I also wonder if it wouldn't make sense to put the access macro in there independent of item type now that we increasingly see books & chapters with DOI.